diff --git a/arch/arm64/boot/dts/sunxi/Makefile b/arch/arm64/boot/dts/sunxi/Makefile
index 5f503d4f7..676fb2570 100644
--- a/arch/arm64/boot/dts/sunxi/Makefile
+++ b/arch/arm64/boot/dts/sunxi/Makefile
@@ -9,7 +9,10 @@ dtb-$(CONFIG_ARCH_SUN50IW1P1) += sun50iw1p1-fpga.dtb  sun50iw1p1-perf1_v1_0.dtb
 dtb-$(CONFIG_ARCH_SUN50IW6P1) += sun50iw6p1-soc.dtb
 
 dtb-$(CONFIG_ARCH_SUN50IW8P1) += sun50iw8p1-fpga.dtb sun50iw8p1-soc.dtb
-dtb-$(CONFIG_ARCH_SUN50IW9P1) += sun50i-h616-orangepi-zero2.dtb
+dtb-$(CONFIG_ARCH_SUN50IW9P1) += sun50i-h616-orangepi-zero2.dtb \
+				sun50i-h616-t95.dtb \
+				sun50i-h313-x96q-1g-8g.dtb \
+				sun50i-h313-x96q.dtb
 dtb-$(CONFIG_ARCH_SUN50IW10P1) += board.dtb
 dtb-$(CONFIG_ARCH_SUN50IW5T) += sun50iw5t-fpga.dtb
 
